src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/Table.java

```diff
@@ -331,14 +332,36 @@ public void addRow(T item, List<Content> contents) {
             }
         }
         int colIndex = 0;
+        String regex = "<(?:a|area|button|input|object|select|textarea)\\b";
+        Pattern pattern = Pattern.compile(regex);
         for (Content c : contents) {
             HtmlStyle cellStyle = columnStyles.get(colIndex);
+            /**
+             * The attribute tabindex="0" allows a non-form/link/object element to be placed
+             * in the default navigation order, making any element focusable and capable of triggering interaction.
+             *
+             * In this context, we add tab order only to plain text elements. This is done to
+             * avoid potential violations of the `widget_tabbable_single` rule, which states
+             * that a component with a "tablist" role should not have more than one tabbable element.
+             *
+             * For more information on the `widget_tabbable_single` rule, please refer to the following documentation:
+             * <a href="https://able.ibm.com/rules/archives/latest/doc/en-US/widget_tabbable_single.html#%7B%22message%22%3A%22Component%20with%20%5C%22tablist%5C%22%20role%20has%20more%20than%20one%20tabbable%20element%22%2C%22snippet%22%3A%22%3Cdiv%20tabindex%3D%5C%220%5C%22%20role%3D%5C%22tablist%5C%22%20class%3D%5C%22col-second%20odd-row-color%20method-summary-table%20method-summary-table-tab2%20method-summary-table-tab3%5C%22%3E%22%2C%22value%22%3A%5B%22VIOLATION%22%2C%22POTENTIAL%22%5D%2C%22reasonId%22%3A%22fail_multiple_tabbable%22%2C%22ruleId%22%3A%22widget_tabbable_single%22%7D">widget_tabbable_single</a>
+             */
```

Comment:
I think this new comment is a bit excessive. Can it be boiled down to a single sentence, maybe one or two lines? Also, there's no need to use `/**` doc comment style here. For example (but feel free to improve):

    // Only make element tabbable if it doesn't contain tabbable content.

Regarding the link, I don't think the content after the `#` is necessary. I would prefer the following form:

    // https://able.ibm.com/rules/archives/latest/doc/en-US/widget_tabbable_single.html

Reply:
I simplified comment as per your suggestion. Thank You.

Reply:
Also, links containing `latest` are typically unstable